server/semantics-service/src/main/java/edu/illinois/ncsa/incore/service/semantics/model/Column.java

```diff
@@ -31,7 +32,7 @@ public Column(String name, String titles, String datatype, String description, S
     }
 
     public String getName() {
-        return this.name;
+        return name;
```

Comment:
any reason not return "this.name"?

Reply:
Ah I also noticed for unit, description, required they don't have "this"...

Reply:
I was trying to make the setters and getters consistent, but missed some. I just made the changes in the commits. Thanks!